components/ActivityIndicator/examples/ActivityIndicatorExample.swift

```diff
@@ -0,0 +1,85 @@
+   static let green: UIColor = UIColor(colorLiteralRed: 0.298, green: 0.686, blue: 0.314, alpha: 1.0)
+   static let yellow: UIColor = UIColor(colorLiteralRed: 1.0, green: 0.922, blue: 0.231, alpha: 1.0)
+}
+
+class ViewController: UIViewController {
+   
+   override func viewDidLoad() {
+      super.viewDidLoad()
+      // Do any additional setup after loading the view, typically from a nib.
+      
+      
+      //Initialize single color progress indicator
+      let frame1: CGRect = CGRect(x: view.bounds.width / 2 - 16, y: view.bounds.height / 2 - 116, width: 32, height: 32)
+      let activityIndicator1 = MDCActivityIndicator(frame: frame1)
+      view.addSubview(activityIndicator1)
+      activityIndicator1.delegate = self
+      // Set the progress of the indicator
+      activityIndicator1.progress = 0.6
+      // Set the progress indicator to determinate
+      activityIndicator1.indicatorMode = .determinate
+      activityIndicator1.sizeToFit()
+      activityIndicator1.startAnimating()
+      
+      // Initialize indeterminate indicator
+      let frame2: CGRect = CGRect(x: view.bounds.width / 2 - 16, y: view.bounds.height / 2 - 16, width: 32, height: 32)
+      let activityIndicator2 = MDCActivityIndicator(frame: frame2)
+      view.addSubview(activityIndicator2)
+      activityIndicator2.delegate = self
+      activityIndicator2.indicatorMode = .indeterminate
+      activityIndicator2.sizeToFit()
+      activityIndicator2.startAnimating()
+      
+      // Initialize multiple color indicator
+      let frame3: CGRect = CGRect(x: view.bounds.width / 2 - 16, y: view.bounds.height / 2 + 84, width: 32, height: 32)
+      let activityIndicator3 = MDCActivityIndicator(frame: frame3)
+      view.addSubview(activityIndicator3)
+      // Pass colors you want to indicator to cycle through
+      activityIndicator3.cycleColors = [MDPalette.blue, MDPalette.red, MDPalette.green, MDPalette.yellow]
+      activityIndicator3.delegate = self
+      activityIndicator3.indicatorMode = .indeterminate
+      activityIndicator3.sizeToFit()
+      activityIndicator3.startAnimating()
+      
+      // Initialize with different radius and stroke with
+      let frame4: CGRect = CGRect(x: view.bounds.width / 2 - 24, y: view.bounds.height / 2 + 176, width: 48, height: 48)
+      let activityIndicator4 = MDCActivityIndicator(frame: frame4)
+      view.addSubview(activityIndicator4)
+      activityIndicator4.delegate = self
+      // Set the radius of the circle
+      activityIndicator4.radius = 18.0
+      activityIndicator4.indicatorMode = .indeterminate
+      // Set the width of the ring
+      activityIndicator4.strokeWidth = 4.0
+      activityIndicator4.sizeToFit()
+      activityIndicator4.startAnimating()
+   }
+   
+   
+}
+
+extension ViewController : MDCActivityIndicatorDelegate {
+   func activityIndicatorAnimationDidFinish(_ activityIndicator: MDCActivityIndicator) {
+      return
+   }
+}
```

Comment:
We need to implement catalogBreadcrumbs so the view controller will show up in the catalog.

If you add the following, your view controller should be visible under the catalog activity indicator tile.

// MARK: Catalog by convention
extension ViewController {
  class func catalogBreadcrumbs() -> [String] {
    return ["Activity Indicator", "Activity Indicator (Swift)"]
  }
}